### PR TITLE
fix typo in the documentation comment.

### PR DESCRIPTION
**Pull request by KengoTODA**
I've signed CLA, but I haven't got response from Google yet. Maybe that system doesn't send a confirming report I guess.

I couldn't reopen #321, so I recreated same one. Please check this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,7 @@
 
 # python
 *.pyc
+
+# OSX
+.DS_Store
+
```
Comment (nanaze) [inline .gitignore]: please remove this change from the branch. if you would like to submit it, send as separate PR

**Comment by KengoTODA** [comment]
Thanks!
